docs/tutorials/cli.mdx

```diff
@@ -0,0 +1,173 @@
+---
+title: Getting Started with the CLI
+---
+
+import CLITabs from '@theme/CLITabs';
+import TabItem from '@theme/TabItem';
+
+The [Svix CLI](https://github.com/svix/svix-cli) is one of the fastest ways to interact with our API.
+
+With the Svix CLI you can interact with our API, validate webhook payloads, quickly open our docs, and much more.
+
+In this tutorial, we will learn how to install the Svix CLI and use it to send your first webhook message!
+
+## Install the Svix CLI
+
+<CLITabs>
+<TabItem value="homebrew">
+
+To install the Svix CLI with [Homebrew](https://homebrew.sh), run:
+
+
+
+```sh
+brew install svix/svix/svix
+```
+</TabItem>
+<TabItem value="scoop">
+
+To install the Svix CLI with [Scoop](https://scoop.sh/), run:
+
+
+```sh
+scoop bucket add svix https://github.com/svix/scoop-svix.git
+```
+
+```sh
+scoop install svix
+```
+
+</TabItem>
+<TabItem value="macos">
+
+To install the Svix CLI on macOS without Homebrew:
+
+1. Download the latest Darwin tar.gz file from <https://github.com/svix/svix-cli/releases/latest>
+
+2. Extract the file: `tar -xvf svix_X.X.X_Darwin_x86_64.tar.gz`
+
+2. Optionally, install the binary in a location where you can execute it globally (e.g., /usr/local/bin).
+
+</TabItem>
+<TabItem value="linux">
+
+To install the Svix CLI on Linux without a package manager:
+
+1. Download the latest Linux tar.gz file from <https://github.com/svix/svix-cli/releases/latest>
+
+2. Extract the file: `tar -xvf svix_X.X.X_Linux_x86_64.tar.gz`
+
+3. Run the executable: `./svix`
+
+</TabItem>
+<TabItem value="windows">
+
+To install the Svix CLI on Windows without Scoop:
+
+1. Download the latest Windows tar.gz file from <https://github.com/svix/svix-cli/releases/latest>
+
+2. Extract the `svix_X.X.X_Windows_x86_64.tar.gz` file.
+
+3. Run the `.exe` file
+
+</TabItem>
+</CLITabs>
+
+## Pair with your Svix Account
+
+After installing the Svix CLI, you must pair it with your Svix account.
+
+To do so, run `svix login` in the terminal. You will be prompted for your Svix Auth Token.
+
+Once supplied, it will be stored in your configuration file.
+
+If you prefer not to store your Auth Token on disk, you can temporarily set it via the `SVIX_AUTH_TOKEN` env variable:
+
+ - On macOS and Linux: `export SVIX_AUTH_TOKEN=<MY-AUTH-TOKEN>`
+ - On Windows via command prompt: `set SVIX_AUTH_TOKEN=<MY-AUTH-TOKEN>`
+ - On Windows via PowerShell: `$env:SVIX_AUTH_TOKEN = '<MY-AUTH-TOKEN>'`
+
+## Create an application
+
+You’re now ready to use the Svix CLI. To get started let's create a new application:
```

Comment:
Are we okay with assuming users understand the basics around what "application" means here? I think we should at least link to the getting started docs. Maybe also add a short TL;DR just so we don't link the reader away.

Something like?

```suggestion
You’re now ready to use the Svix CLI. To get started, let's create a new application. This will serve as a hub for all the webhooks sent to one of your clients.
```

Reply:
@tasn thoughts? I don't think everyone's use case is app per client so this may confuse some people

Reply:
I could have sworn I replied! I think we should just link at the top to the quickstart for now and assume they've used the service. Maybe "This tutorial assumes you have already used the service or at least read the [quickstart](...)", or something.

Reply:
done